README.md

```diff
@@ -53,7 +56,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(1), because the amount of memory used at any given operation is simply checking if 2 points are the same, so scaling the sentence does not increase the amount of things to check for each operation, therefore memory is constant.
```

Comment:
# 0

Which two points? Because it you're assuming that the parser has the whole string in memory, then that's automatically O(n).